fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -2504,12 +2507,12 @@ private DataDescription getDataDescription(StarRocksParser.DataDescContext conte
         List<String> colList = null;
         if (context.colList != null) {
             List<Identifier> identifiers = visit(context.colList.identifier(), Identifier.class);
-            colList = identifiers.stream().map(Identifier::getValue).map(String::toLowerCase).collect(toList());
+            colList = identifiers.stream().map(Identifier::getValue).collect(toList());
```

Comment:
Why do this modification alone, is there a bug here?

Reply:
this is another modification which is bug which was fixed and merged 4 days ago ,  when I made this pr , some conflicts resolution staff to modification these lines. I will try to make it was to be